dashboard/test/lib/services/lti_test.rb

```diff
@@ -0,0 +1,60 @@
+require 'test_helper'
+require 'user'
+require 'authentication_option'
+require 'policies/lti'
+# require 'pry'
+
+class Services::LtiTest < ActiveSupport::TestCase
+  id_token = {
+    sub: '12345-a314-4215-b03e-58fe1bd3c8b0',
+    aud: '10000000000001',
+    iss: 'https://some-lms.com',
+    name: 'Han Solo',
+    given_name: 'Han',
+    family_name: 'Solo',
+    'https://purl.imsglobal.org/spec/lti/claim/roles' => [
+      'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Administrator',
+      'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Instructor',
+      'http://purl.imsglobal.org/vocab/lis/v2/membership#Instructor',
+      'http://purl.imsglobal.org/vocab/lis/v2/system/person#SysAdmin',
+      'http://purl.imsglobal.org/vocab/lis/v2/system/person#User',
+    ],
+  }
+
+  student_id_token = {
```

Comment:
There's a lot of repeated set up of test id_tokens. It would be nice to define these once and reference them in our different test files.

Reply:
I'm going to move this into a factory. We can have a default `lti_id_token` defined, and override it for a student (ie. remove/modify the roles)

Reply:
So it turns out factories won't work here. Factories are for setting up an instance of a model or a class for testing. When you try to create a factory for just a plan old object/hash, you get an error saying it is an undeclared constant (or something like that)